packages/app-project/src/screens/ClassifyPage/components/YourStats/components/DailyClassificationsChart/DailyClassificationsChart.js

```diff
@@ -1,23 +1,112 @@
 import counterpart from 'counterpart'
-import PropTypes from 'prop-types'
+import { array, object, string } from 'prop-types'
 import React from 'react'
+import { withTheme } from 'styled-components'
+import { AxisBottom, AxisLeft } from '@vx/axis'
+import { Group } from '@vx/group'
+import { Bar } from '@vx/shape'
+import { scaleBand, scaleLinear } from '@vx/scale'
+import WidgetHeading from '../../../../../../shared/components/WidgetHeading'
 
 import en from './locales/en'
 
 counterpart.registerTranslations('en', en)
 
-function DailyClassificationsChart () {
+function DailyClassificationsChart ({ counts, projectName, theme }) {
+  const HEIGHT = 200
+  const PADDING = 20
+  const WIDTH = 300
+  const xScale = scaleBand({
+    rangeRound: [0, WIDTH],
+    domain: counts.map(count => count.longLabel),
+    padding: 0.1
+  });
+
+  const yScale = scaleLinear({
+    rangeRound: [HEIGHT - PADDING, 0],
+    domain: [0, Math.max(...counts.map(count => count.count))],
+    nice: true
+  });
+
+  function tickLabelProps () {
+    return {
+      dx: '-0.25em',
+      dy: '0.2em',
+      fill: theme.global.colors['light-4'],
+      fontFamily: 'Arial',
```

Comment:
You can get the font family from the theme: `theme.global.font.family`. Or would just `inherit` here work?

Reply:
Good question. I'm not sure if those props can inherit document styles.